service/header/service.go

```diff
@@ -44,3 +44,9 @@ func (s *Service) Stop(context.Context) error {
 	log.Info("stopping header service")
 	return nil
 }
+
+// TODO(@Bidon15): This has to be a subscription
```

Comment:
I think we can just introduce a new method for this. `IsSyncing` is a valuable endpoint for RPC, so we should leave this one as is and implement a new method that will give you the functionality you need.

Reply:
Ok. I will separate this todo from this method